department-of-veterans-affairs/abd-vro
# Script fix

## What was the problem?
Previously, we created the truststore using the `VA-Internal-S2-RCA1-v1.cer` and `VA-Internal-S2-ICA4.cer` files, but encountered certificate issues in dev environment. After debugging and validating with curl commands, we determined that `AllVAInternalCAs.p7b` is the right file.

Associated tickets or Slack threads:
- #?

## How does this fix it?[^1]
script change to use `AllVAInternalCAs.p7b` file

## How to test this PR
- run the script and checkout `output.json` file in the same folder


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
